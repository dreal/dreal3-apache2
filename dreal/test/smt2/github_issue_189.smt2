;; From https://github.com/dreal/dreal4/issues/188
(set-logic QF_NRA)
(set-option :worklist-fixpoint true)
(declare-fun _substvar_82_ () Bool)
(declare-fun _substvar_91_ () Bool)
(declare-fun _substvar_81_ () Bool)
(declare-fun _substvar_80_ () Bool)
(declare-const v2 Bool)
(declare-const r4 Real)
(check-sat)
(push 1)
(assert false)
(check-sat)
(pop 1)
(assert (>= 55596923.0 0.3719817))
(assert (=> _substvar_91_ (<= 0.3719817 r4)))
(check-sat)
