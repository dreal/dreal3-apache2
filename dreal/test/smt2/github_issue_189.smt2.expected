delta-sat with delta = 0.001
unsat
delta-sat with delta = 0.001
